controllers/apply/awards-for-all/fields.js

```diff
@@ -2057,6 +2057,38 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                 }
             ]
         },
+        mainContactIsValid: {
```

Comment:
This flag needs to be wrapped around this entry too. The validation schema is built from the `allFields` object, so this will force this field to be required even if the field isn't shown unless we wrap this in a flag.